tests/test_shared.py

```diff
@@ -147,3 +149,12 @@ def test_state_dict_too_large(local_remote_dir: Tuple[str, str]):
 
     with pytest.raises(ValueError, match='The StreamingDataset state dict*'):
         dataset.load_state_dict(big_state_dict)
+
+
+@pytest.mark.parametrize('dtype', [np.int32, np.int64, np.float32, np.float64])
+@patch('streaming.base.shared.array.SharedMemory')
+def test_shared_array_size_is_integer(mock_shared_memory, dtype):
+    SharedArray(3, dtype=dtype, name='test_shared_array')
```

Comment:
@bchiang2 Looks like there are some linting issues here, see prev comment for the error. You should be able to run `pip install pre-commit` and then run linting for your PR locally before pushing the changes!

Reply:
On a side note, it might be helpful to use docker container could simplify the process of setup and running `pre-commit run --all-files` for other developers.

Reply:
Oh I see, thanks for pointing this out! A docker container would be a good idea, let me bring that up with the team.